pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -860,6 +860,43 @@ int socket__http2_headers_parser(struct __sk_buff *skb) {
     return 0;
 }
 
+SEC("socket/http2_dynamic_table_cleaner")
```

Comment:
After getting the dispatcher_args_copy the all logic is the same, (except of the next call). Should we move it the common to reduce code duplication?

Reply:
I think that's something we should do, but no in the scope of this PR